modules/virt-deprecated-tasks.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="virt-deprecated-tasks_{context}"]
+= Removing deprecated or unused resources
+
+* If you no longer use Tekton (TTO) in your environment after upgrading to OpenShift Container Platform 4.14 or later, you can remove any remaining Tekton-related resources that were previously managed by the ssp-operator.
```

Comment:
- tekton-task-operator was deprecated in 4.14 and remained in the code and supported until it was removed from the code in v4.17.
- Please do not abbreviate Tekton as TTO. We never used the acronym in our documentation. The documentation always spelled-out tekton-tasks-operator.
- Best practice: Keep sentences less than 30 words.
- Best practice, when starting a sentence with if, then include the "then" after the comma.

Recommend re-writing:

The tekton-tasks-operator was deprecated and removed from OpenShift Virtualization 4.17. You can remove any remaining resources that were used by the tekton-tasks-operator and pipelines by running the following command.